stable/appmesh-manager/README.md

```diff
@@ -0,0 +1,183 @@
+# App Mesh Manager
+
+App Mesh manager Helm chart for Kubernetes
+
+## Prerequisites
+
+* Kubernetes >= 1.13
+* IAM policies
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "appmesh:*",
+                "servicediscovery:CreateService",
+                "servicediscovery:GetService",
+                "servicediscovery:RegisterInstance",
+                "servicediscovery:DeregisterInstance",
+                "servicediscovery:ListInstances",
+                "servicediscovery:ListNamespaces",
+                "servicediscovery:ListServices",
+                "servicediscovery:GetOperation",
+                "servicediscovery:GetInstancesHealthStatus",
+                "servicediscovery:UpdateInstanceCustomHealthStatus",
+                "route53:GetHealthCheck",
+                "route53:CreateHealthCheck",
+                "route53:UpdateHealthCheck",
+                "route53:ChangeResourceRecordSets",
+                "route53:DeleteHealthCheck"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+## Installing the Chart
+
+Add the EKS repository to Helm:
+
+```sh
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+Install the App Mesh CRDs:
+
+```sh
+kubectl apply -k github.com/aws/eks-charts/stable/appmesh-manager//crds?ref=master
```

Comment:
nit: double "//"

Reply:
This is not a typo, that’s how you tell kustomize to fetch only the crds dir.

Reply:
Ah thanks for clarifying